assets/js/translator.js

```diff
@@ -1242,6 +1265,36 @@ function autoSelectDstLang() {
     $('#dstLangSelect').val(curDstLang).change();
 }
 
+// Check if the Language pair Exist or Not
```

Comment:
These three functions can go inside the separate function you create.

Reply:
Done :)